cmd/dbbrowser/main.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/medyagh/gopogh/pkg/db"
+)
+
+var dbPath = flag.String("db_path", "", "path to cloudsql db in the form of 'host=HOST_NAME user=DB_USER dbname=DB_NAME password=DB_PASS'")
+
+func main() {
+	flag.Parse()
+	if *dbPath == "" {
+		log.Fatal(fmt.Errorf("db_path not specified"))
+	}
+	cfg := db.Config{
+		Type: "cloudsql",
+		Path: *dbPath,
+	}
+	pg, err := db.NewCloudPostgres(cfg)
```

Comment:
This is a little confusing since the type is hardcoded to `cloudsql` but then we call `db.NewCloudPostgres`. Especially when looking at `pkg/db/db.go` line 24 and can see they're explicitly different types.

```
	switch cfg.Type {
	case "sqlite":
		return newSQLite(cfg)
	case "postgres":
		return newPostgres(cfg)
	case "cloudsql":
		return newCloudSQL(cfg)
	default:
		return nil, fmt.Errorf("unknown backend: %q", cfg.Type)
	}
```

Reply:
cloudsql isn't a database by itself (it can be mysql or postgres). In this case newCloudSQL creates a datab (which in reality is just a Postgres instance but held as a datab). Should I refactor it so that the cloudsql type is now cloudpostgresql and so we can use an interface function to print with the correct schema because currently if we use newCloudSQL we get a datab that doesn't have specific information on what type of database it is?